### PR TITLE
add 157-date-format test

### PR DESCRIPTION
**Pull request by cpsievert**
See https://github.com/rstudio/shiny/pull/2403

### CODE REVIEW

**Comment by cpsievert** [comment]
Use `renderText()` instead (capture all the warnings in a character vector and print the str)

**Comment by cpsievert** [comment]
Use similar design to 152

**Comment by wch** [comment]
https://github.com/rstudio/shiny-examples/blob/master/152-set-reactivevalue/app.R

File: 157-date-format/app.R [hunk 1]
```diff
@@ -1,63 +1,68 @@
...
+  )
 )
 
 
 server <- function(input, output, session) {
 
-  # number of warnings thrown over the course of one tick
-  nWarnings <- 0
+  # variable for tracking warning message
+  warn_messages <- NULL
```
Comment (wch) [inline 157-date-format/app.R]: This should be `reactiveVal(NULL)` to ensure that `output$warnings` and `output$res` execute after `output$inputs` re-executes.

**Comment by cpsievert** [comment]
@wch good to merge?

**Pull request merged**